test/mpi/test_mpi.h

```diff
@@ -277,6 +281,8 @@ class TestAllgather : public TestCase {
 };
 
 class TestAllgatherv : public TestCase {
+    ucc_mc_buffer_header_t *counts_header;
```

Comment:
Why do we need mc_alloc for counts and displacements at first place? We can always just use ucc_malloc, because counts and displacements are HOST(CPU) arrays

Reply:
@vspetrov it's like that in many gtests, maybe need another PR to change these